utils/config/SConscript

```diff
@@ -5,7 +5,8 @@ def scons():
     """Execute build"""
     Import('env')
 
-    env.Install("$PREFIX/etc", ['daos_server.yml', 'daos.yml'])
+    env.Install("$PREFIX/etc", ['daos_server.yml', 'daos.yml',
```

Comment:
if we have daos_agent shouldn't we change daos to daos_dmg? not for this patch just a thought